Assignment2/guilledmv-FromRDF-JSONLD.jsonld

```diff
@@ -0,0 +1,61 @@
+{
+	"@context":{
+
+		"xsd":"http://www.w3.org/2001/XMLSchema#",
+		"components":"http://example.org",
+		"Class01":"components:Class01",
+		"Sensor029":"Class01:Sensor029",
+		"Computer101":"Class01:Computer101",
+		"Measurement8401":"Sensor029:Measurement8401",
+		"User10A":"Computer101:User10A",
+		"includes":"components:includes",
+		"hasMeasurement":"components:hasMeasurement",
+		"hasTemperature":"components:hasTemperature",
+		"atTime":"components:atTime",
+		"hasOwner":"components:hasOwner",
+		"hasName":"components:hasName"
+
+	},
+	"@id":"http://example.org/Class01"
+	"@type":"components:Class01",
+	"includes": [
+	{
+		"Sensor029":{
```

Comment:
"Sensor029": isn't needed. I suggest you remove it. Also, delete "Measurement8401":, "Computer101":, and "User10A":
Be careful with the curly braces.